tests/test_controlnet_aux.py

```diff
@@ -119,3 +119,9 @@ def test_shuffle(img):
 def test_zoe(img):
     zoe = ZoeDetector.from_pretrained("lllyasviel/Annotators")
     common("zoe", zoe, img)
+
+def test_dwpose(img, det_config, det_ckpt, pose_config, pose_ckpt):
```

Comment:
Does this work?  I don't think you can have extra parameters besides img.

Reply:
Fixed.